### PR TITLE
[BXMSDOC-3768] adding usage documentation

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

**Comment by ge0ffrey** [comment]
For me this looks good to merge.
Style wise, there's a lot of usage of second person [1] (instead of active voice [2]), I remember Gemma worked hard in optaplanner-docs to standarize on active voice as much as possible. @vidya-iyengar What's the policy on that these days?

@Christopher-Chianelli Please review this PR and merge it if there are no blocking comments.

[1] https://developers.google.com/style/person
[2] https://developers.google.com/style/voice

File: employee-rostering-docs/src/main/asciidoc/Using/er-availability-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+.Procedure
+. To view and edit employee availability, click the *Availability Roster* tab.
++
+In the top left part of the window, you can see the dates for which the roster is displayed. To view other weeks, you can use the *<* and *>* buttons next to the *Week of* field. Alternatively, you can click the date field and change the date to view the week that includes this date.
++
+You can see the numbers of currently visible employees in the top right part of the browser window, for example, `1-10 of 34`. You can use the *<* and *>* buttons next to the numbers to display other employees in the list. 
++
+. To create an availability entry for an employee, click empty space on the intersection of a date (column) and an employee (row). Initially, an `Unavailable` entry for the entire day is created.
+. To change an availability entry, double-click the entry. You can change the following settings:
```
Comment (Christopher-Chianelli) [inline employee-rostering-docs/src/main/asciidoc/Using/er-availability-proc.adoc]: You just need to single click

File: employee-rostering-docs/src/main/asciidoc/Using/er-availability-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+** *From* and *To* date and time: The time span to which the availability entry applies.
+** Status: you can select *Unavailable*, *Desired*, or *Undesired* status from a drop-down list.
++
+To save the entry, click *Apply*.
+. To delete an availability entry, double-click the entry, then click *Delete availability*.
+
```
Comment (Christopher-Chianelli) [inline employee-rostering-docs/src/main/asciidoc/Using/er-availability-proc.adoc]: An additional, quicker way to update the availability type is to use the "cross/x/checkmark" buttons that appears when you hover over an availability entry. The cross/left one (O with a \) is unavailable, the x/center one (O with a X) is undesired and the checkmark/right one (O with a checkmark) is desired. There also a X on the top-right corner that allows you to delete the availability.

File: employee-rostering-docs/src/main/asciidoc/Using/er-availability-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+IMPORTANT: If an employee is already assigned to a shift and then you create or change an availability entry during this shift, the assignment is not changed automatically. You must create the employee roster again to apply new or changed availability entries.
```
Comment (Christopher-Chianelli) [inline employee-rostering-docs/src/main/asciidoc/Using/er-availability-proc.adoc]: Should be "shift roster" not "employee roster"

File: employee-rostering-docs/src/main/asciidoc/Using/er-employees-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,20 @@
+[id='er-employees-proc']
+= Entering the list of employees
+
+You must enter the list of all employees of the business and the skills they possess. The application rosters these employees to spots according to their skills.
+
+.Procedure
+
+. To enter or change the list of employees, click the *Spots* tab.
```
Comment (Christopher-Chianelli) [inline employee-rostering-docs/src/main/asciidoc/Using/er-employees-proc.adoc]: Should be the *Employees* tab

**Comment by ge0ffrey** [comment]
@mramendi Can you take a look at christophers suggestions above? ^

**Comment by triceo** [comment]
`optaweb-employee-rostering` project is no longer maintained, closing this PR.

**Pull request closed without merging**